spec/schemas/_common.yaml

```diff
@@ -1814,39 +1814,39 @@ components:
           description: The total amount of time for all shard scroll operations.
           $ref: '#/components/schemas/Duration'
         scroll_time_in_millis:
-          description: The total amount of time for all shard scroll operations, in milliseconds.
+          description: The total amount of time for all shard scroll operations, in milliseconds. 
           $ref: '#/components/schemas/DurationValueUnitMillis'
         scroll_total:
           description: The total number of shard scroll operations.
           type: number
         point_in_time_total:
-          description: The total number of shard Point in Time (PIT) contexts that have been created (completed and active) since the node last restarted.
+          description: The total number of shard Point in Time (PIT) contexts that have been created (completed and active) since the node last restarted. 
```

Comment:
```suggestion
          description: The total number of shard Point in Time (PIT) contexts created (completed and active) since the node last restarted. 
```